helm/templates/_helpers.tpl

```diff
@@ -30,3 +30,21 @@ If release name contains chart name it will be used as a full name.
 {{- define "service-account.name" -}}
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
+
+{{/* Import or generate certificates for webhook */}}
+{{- define "aws-gateway-controller.webhookTLS" -}}
+{{- if (and .Values.webhookTLS.caCert .Values.webhookTLS.cert .Values.webhookTLS.key) -}}
+caCert: {{ .Values.webhookTLS.caCert }}
+cert: {{ .Values.webhookTLS.cert }}
+key: {{ .Values.webhookTLS.key }}
+{{- else -}}
```

Comment:
Do we need the `keepTLSSecret` option? saw alb controller has that one https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/3ad23913c416258f37da9b6e6e248aa984a9f183/helm/aws-load-balancer-controller/values.yaml#L220

Reply:
It's something we can consider adding? For now if folks use the auto-installed cert the new one will be just as good. If they are using their own cert the same values for ca/cert/key should work across installs.